src/elona/lua_env/lua_class/lua_class_character.cpp

```diff
@@ -653,6 +693,17 @@ void LuaCharacter::bind(sol::state& lua)
     LuaCharacter.set(
         "quality", LUA_API_ENUM_PROPERTY(Character, quality, Quality));
 
+    /**
+     * @luadoc proto field table
+     *
+     * [R] The prototype data of the character.
+     */
+    LuaCharacter.set("proto", sol::property([](Character& self) {
```

Comment:
I prefer `prototype` to `proto`. For programmers (or maybe English speaker?), it is obvious that "proto" stands for "prototype", but for non-engineers who speak Japanese, it may be not clear. It would be better that you can search the words in Google and get Japanese (or other non-English) translation with ease. Thus, generally speaking, Lua interface should use full spellings except for some words very commonly known like HP/MP, ID, PV/DV, etc. In the future,  other abbreviations such as "chara" and "pos" can be changed.